README.md

```diff
@@ -25,9 +25,131 @@ This requires having [Luma3DS](https://github.com/AuroraWright/Luma3DS) on your
 
 ## Building the patches for custom save paths
 
-If you want a different game Id or save path, run these commands (replace `SD_SAVE_PATH` and `GAME_ID`):
+### Requirements
 
-```
-armips transporter.s -strequ SD_SAVE_PATH "/roms/nds/saves/black.sav" -strequ GAME_ID "IRBO"
-flips -c code.bin code_patched.bin code.ips
-```
+- [ ] The `.code` file of the Dream Radar or Pokémon Transporter application (see [below](#dump) how to dump it)
+- [ ] A copy of this repository (or at least of the respective `.s` file of the title you want to modify)
+- [ ] A copy of [armips](https://github.com/Kingcom/armips)
+- [ ] A copy of [flips (Floating IPS)](https://github.com/Alcaro/Flips)
+
+<details>
+   <summary>
+   <h3>
+      <a name="dump">Dumping the <code>.code</code></a>
+   </h3>
+   </summary>
+
+#### Requirements
+
+- [ ] The respective title (Dream Radar or Transporter) installed on your 3DS
+- [ ] [GodMode9](https://github.com/d0k3/GodMode9) on your 3DS (should be installed if you followed [this guide](https://3ds.hacks.guide/))
+
+<h4>Dumping</h4>
+
+1. Boot into GodMode9 (if you followed the aforementioned guide, hold start while booting)
+2. Open the title manager:
+
+   - Navigate to `[Y:] TITLE MANAGER` and press <kbd>A</kbd>
+   <br>or<br>
+   - Navigate to `[A:] SYSNAND SD` and press <kbd>R + A</kbd> for drive options and select `Open title manager` on the bottom screen by using the D-pad and then press <kbd>A</kbd>
+3. Find the entry of the title you want to dump and press <kbd>A</kbd>, it should have the same folder name as above:
+
+   - Pokémon Transporter: `00040000000C9C00`
+   - Japanese Pokémon Dream Radar: `0004000000073200`
+   - All Regions Pokémon Dream Radar: `00040000000AE100`
+
+4. Select `Open title folder` and press <kbd>A</kbd>
+5. Select the `.tmd` file and press <kbd>A</kbd>
+6. Select `TMD file options...` and press <kbd>A</kbd>
+7. Select `Mount CXI/NDS to drive` and press <kbd>A</kbd>
+8. Press <kbd>A</kbd> to enter the mount point
+9. Select the `exefs` folder and press <kbd>A</kbd>
+10. Select the `.code` file and press <kbd>A</kbd>
+11. Select `Copy to 0:/gm9/out` and press <kbd>A</kbd>
+12. Press <kbd>A</kbd> to continue
+
+Success! The dumped `.code` is now stored on your SD card at `SD:/gm9/out`. Copy this file over to the repository folder on your computer using your preferred method, e.g. with [FTPD](https://github.com/mtheall/ftpd) or by plugging in your SD into your computer. Remember that on Unix and Unix-like environments files with filenames starting with a `.` are treated as hidden files, so make sure your preferred file browser shows them when trying to transfer the file.
+</details>
+
+### Building the patches yourself
```

Comment:
It seems like this line should be the parent of the following guides, however this and the guides are each `h3`, and `h4` feels a bit too small/nested for the following guides.

Let's remove this "Building the patches yourself" line and use "Build Patch for Dream Radar (All regions)", "Build Patch for Dream Radar (Japan)", and "Build Patch for Transporter" for the guides beneath here.  That will allow each guide to be `h3` without losing any context.